docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -26,35 +26,41 @@ weight: 80
 
 # Configure data links
 
-Data links allow you to provide more granular context to your links. You can create links that include the series name or even the value under the cursor. For example, if your visualization showed four servers, you could add a data link to one or two of them.
+Data links allow you to provide more granular context to your links. You can create links that include the series name or even the value under the cursor. For example, if your visualization shows four servers, you can add a data link to one or two of them. You can also link panels using data links.
 
-The link itself is accessible in different ways depending on the visualization. For the Graph you need to click on a data point or line, for a panel like
-Stat, Gauge, or Bar Gauge you can click anywhere on the visualization to open the context menu.
+The link itself is accessible in different ways depending on the visualization. For the time series visualization you need to click a data point or line:
 
-You can use data link variables or data links to create links between panels. You can use variables in data links to send people to a detailed dashboard with preserved data filters. For example, you could use variables to specify a time range, series, and variable selection. For more information, refer to [Data link variables](#data-link-variables).
+![Time series visualization with a data link displayed](screenshot-time-series-data-link-v10.3.png)
+
+For visualizations like stat, gauge, or bar gauge you can click anywhere on the visualization to open the context menu:
+
+![Stat visualization with a data link displayed](screenshot-stat-data-link-v10.3.png)
+
+If there's only one data link in the visualization, clicking anywhere on the visualization opens the link rather than the context menu.
 
 ## Data link variables
 
-You can use variables in data links to refer to series fields, labels, and values. For more information about data links, refer to [Data links](#data-links).
+Variables in data links let you send people to a detailed dashboard with preserved data filters. For example, you could use variables to specify a label, time range, series, or variable selection.
 
-To see a list of available variables, type `$` in the data link **URL** field to see a list of variables that you can use.
+To see a list of available variables, enter `$` in the data link **URL** field.
 
```

Comment:
**TO DATAVIZ REVIEWERS**: When you enter the $ to see variable options, they come up in plain text first and then when you select them, you see them the way we have them in the tables below. I tried to match up those plain text UI labels with the variables below, but I started to not be able to match them up. Is that something we should try to do? Kind of like:
| UI selection | Variable           | Description                                                         |
| ------------ | ------------------ | ------------------------------------------------------------------- |
| Time range  | `__url_time_range` | Current dashboard's time range (for example, `?from=now-6h&to=now`) |

Reply:
If I'm understanding correctly, then this makes a lot of sense to me. Would definitely be helpful to have the mapping there so folks understand the option 😄

Reply:
Okay, I'm going to create a separate PR so as not to block this work.